### ISSUE TITLE
Responsive behavior no longer working

### ISSUE THREAD

**Issue by wfendler** [kenwheeler/slick#1303]
What I need is for this element to be "unslicked" below 900px. Before the update, the following code worked as expected:

```javascript
$('#slick').slick({
  "arrows": false,
  "speed": 1500,
  "swipe": false,
  "cssEase": "cubic-bezier(0.165, 0.840, 0.440, 1.000)",
  "responsive": [{
    breakpoint: 900,
    settings: "unslick"
  }]
});
```

When the page loads at >900px the carousel works. I size the window below 900px and it "unslicks". But where it fails now is when I go from <900px to >900px. The carousel no longer reinitializes. Am I using the `responsive` property incorrectly? How might I achieve what I had working previously?

**Comment by nightbook** [comment]
I'm experiencing this as well.

**Comment by nightbook** [comment]
I also noticed not only does it not 'reslick' when you go greater than the breakpoint. But when less than the breakpoint and it's been unslicked if you wait a little bit then you'll notice the items get the css classes 'slick-active slick-center' added to them one at a time at the speed you specified.

**Comment by nightbook** [comment]
I've rolled back to 1.5.0 to get the responsive aspect working. But even in that version the additional classes are added so branching into a seperate ticket;
https://github.com/kenwheeler/slick/issues/1304

**Comment by simeydotme** [comment]
__Regarding the classes being applied after unslick__; it was an oversight in the `asNavFor()` method and should soon be resolved by #1306 :) (please feel free to review).

===

__Regarding the unslicked-carousel not becoming slicked__; this is documented (dubiously) in the 1.4.0 tag: https://github.com/kenwheeler/slick/releases/tag/1.4.0 
there's a line in the __responsive__ bullet point stating: 
> Note, you can't window shrink this, once you unslick, you are unslicked.  

which means after `unslick` occurs, you cannot re-slick unless you call the `.slick()` method on the element to re-initialise. :smile: (also mentioned in #1190 and #994)
